api/mesh/v1alpha1/dataplane_insight.proto

```diff
@@ -93,3 +96,20 @@ message DiscoveryServiceStats {
   // Number of xDS responses NACKed by the Dataplane.
   uint64 responses_rejected = 3;
 }
+
+message Version {
```

Comment:
please comment the fields to follow the rest of the .proto convention

Reply:
Ok, I added comments.